WTWSMS/decisions/minor_decisions.txt

```diff
@@ -630,7 +637,7 @@ decisions = {
 				factor = 0.001             # Only a small minority of Christians ever go on a pilgrimage
 			}
 			modifier = {
-				factor = 3
+				factor = 50
```

Comment:
Do the math.

0.001 x 50 = 0.05 .

That means Zealous characters have but a FIVE percent change to go on a Pilgrimage, and only IF they are Zealous.